### ISSUE TITLE
ssh-import-lp-id fails on OSX 10.10.4 when installed from pip

### ISSUE THREAD

**Issue by marcoceppi** [cmars/ssh-import-id#6]
```
thunder:~ marco$ ssh-import-id marcoceppi
Traceback (most recent call last):
  File "/usr/local/bin/ssh-import-id-lp", line 25, in <module>
    from urllib.parse import quote_plus
ImportError: No module named parse
2015-10-05 17:06:39,822 ERROR Error executing protocol helper [/usr/local/bin/ssh-import-id-lp]
```

ssh-import-id==4.1

Even after hacking around a try/except block for urllib.parse to `import urlparse as quote_plus` the script failed further down the line

**Comment by dustinkirkland** [comment]
What is the first line of /usr/local/bin/ssh-import-id?  How is it invoking the interpreter?  ssh-import-id needs python3 to work correctly.  This error indicates that you're running it under python2.7 probably.

**Comment by marcoceppi** [comment]
Well, pip install on a python 2.7 system worked. So, I suppose it needs to
be updated on pypi to reflect that py2 doesn't work?

**Comment by dustinkirkland** [comment]
Sure.

Can you also try "pip3 install ssh-import-id"?

**Comment by marcoceppi** [comment]
This is the top of the file:
```
#!/usr/bin/python
```

Python3 does not come installed on OSX out of the box. Seems it be better to either wrap ssh-import-id in a homebrew formula where proper deps can be installed or support python2 in the codebase.

After installing python3 with homebrew, pip3 install still fails since the interpreter is set to /usr/bin/python. Changing /usr/local/bin/ssh-import-lp-id to `#!/usr/bin/env python3` resolved this, though having py2 and 3 compat seems the best way forward as not even Ubuntu has dropped py2 in a released os version yet.

**Comment by dustinkirkland** [comment]
I'll see what I can do.

Also, could you move this issue over to https://github.com/dustinkirkland/ssh-import-id ?

**Comment by dustinkirkland** [comment]
Or, better yet, bugs.launchpad.net/ssh-import-id which is the upstream bug tracker for this project?

**Comment by marcoceppi** [comment]
It's duplicated here: https://bugs.launchpad.net/ssh-import-id/+bug/1428840

**Comment by dustinkirkland** [comment]
Thanks.  I'm moving the discussion there.

**Comment by dustinkirkland** [comment]
FYI, fix released and uploaded to pypi.  Can you retry?

**Comment by marcoceppi** [comment]
This is working with the standard python2 shipped on 10.10, thanks for the
speedy fix!
